components/faction/commons/faction.lua

```diff
@@ -64,6 +66,36 @@ function Faction.read(faction)
 		or Faction.aliases[faction]
 end
 
+--- Parses multiple factions from input.
+-- Returns a table of the found factions short handle/identifier.
+-- Returns empty table if no input is specified.
+---@props props {input: string, sep: string?}
+---@return table
+function Faction.readMultiFaction(props)
```

Comment:
```suggestion
function Faction.readMultiFaction(input, options)
```

I don't think it makes sense to put `input` and `sep` in a single object here, since the code paths leading up to this will likely only deal with `input`, with `sep` being specified right before the call.

Reply:
If I wanted to add multi-faction support to a module, how would I know which `sep` to use? (Maybe it's out of scope, if this PR is intended to cover existing use cases)